specs/settings-spec.js

```diff
@@ -0,0 +1,94 @@
+function formItemByLabel (label) {
+  return cy
+    .get('label')
+    .contains(label)
+    .parents('.form-item')
+    .find('input')
+}
+
+describe('The Example App', () => {
+  context('Settings', () => {
+    beforeEach(() => {
+      cy.visit('/settings')
+    })
+
+    it('renders setting with default values', () => {
+      cy.title().should('equals', 'Settings — The Example App')
+      cy.get('main h1').should('have.text', 'Settings')
+      cy.get('.status-block')
+        .should('have.length', 1)
+        .invoke('text').then((text) => {
+          expect(text).to.contain('Connected space:')
+        })
+
+      formItemByLabel('Space ID').should('have.value', Cypress.env('CONTENTFUL_SPACE_ID'))
+      formItemByLabel('Content Delivery API - access token').should('have.value', Cypress.env('CONTENTFUL_DELIVERY_TOKEN'))
```

Comment:
This is failing, because as per [the source locale files](https://github.com/contentful/the-example-app-assets/blob/development/src/locales/en-US.json#L86), the correct label is "Content Delivery API access token". Same with the Preview API token.